indexer/altitude_loader.cpp

```diff
@@ -29,8 +29,14 @@ void LoadAndMap(size_t dataSize, ReaderSource<FilesContainerR::TReader> & src, T
 
 namespace feature
 {
-AltitudeLoader::AltitudeLoader(MwmValue const & mwmValue)
+AltitudeLoader::AltitudeLoader(Index const & index, MwmSet::MwmId const & mwmId)
```

Comment:
If you'd like to avoid re-use of handles, pass it by value here, not by const ref. So, it won't be possible to share it. But I'm not insisting.